opendbc/car/honda/carstate.py

```diff
@@ -21,7 +21,7 @@
 
 def get_can_messages(CP, gearbox_msg):
   messages = [
-    ("ENGINE_DATA", 100),
+    ("ENGINE_DATA", 0), # TODO : not on every car (missing from Integra)
```

Comment:
We definitely can't disable the frequency check for other cars, this would have to become a conditional.

Reply:
Certainly not ideal, but given that "STEER_MOTOR_TORQUE" doesn't have a frequency check, I thought it would be better than the additional lines for a conditional check. I can change it though.

Reply:
It's a nice-to-have bit of information on Honda, but isn't used in carcontroller or safety like it is on certain other brands. Our vehicle speed reference is definitely safety critical.